frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/PipelineResources.tsx

```diff
@@ -11,13 +11,14 @@ type PipelineResourcesParam = {
 
 const PipelineResources: React.FC<PipelineResourcesParam> = (props) => {
   const { addLabel = 'Add Pipeline Resources', fieldName, isReadOnly = false } = props;
```

Comment:
Adjust to fit with UX text. Should be singular.

Reply:
updated